### ISSUE TITLE
Screen reader and sapi support

### ISSUE THREAD

**Issue by 76534** [ublue-os/bazzite#2119]
I plan to use this OS on an ROG ally X. The problem is I'm completely blind, and as far as I can tell this does not have a screen reader, whether it be  ORCA, which even on Steam OS only works in the Linux part of the OS, not the gaming mode, and even if it did, if you install a game that requires SAPI for its in Game screen reader it will not work. Is there any way to add these in the future? Regarding SAPI I'm pretty sure it can be installed on proton, if there would be a way to get to communicate with other games and apps that would be nice, if someone could write a guide for this even if I had to use sited assistance just to install it that would be really helpful, but we still need a screen reader to navigate around for example Steam.

**Comment by noelmiller** [comment]
As far as I know, Orca should be shipped with the Bazzite image by default, so you may want to try that for access to Orca.

My suggestion would be to reach out to Valve upstream and see if there are plans to support screen readers in Steam Game Mode, as this is something that would be out of scope for Bazzite.

Sorry we can't be of more help!
